flow/core/kernel/energy.py

```diff
@@ -0,0 +1,119 @@
+import dill as pickle
+import boto3
+import botocore
+import math
+import random
+import statistics
+import numpy as np
+from scipy.interpolate import interp1d
+from collections import namedtuple
+
+
+class EnergyModel():
+    """Base energy model class."""
+
+    def __init__(self,kernel):
+        self.vehicle = vehicle
+
+    def get_energy(self, veh_id):
+        #return self.calc_energy(veh_id)
+        pass
+
```

Comment:
i think this might be a good place to implement common unit conversions that will be used by the child classes, like joules <-> megajoules, soc <-> joules, gallons <-> joules, etc.